packages/ui/atoms/Button/Button.twig

```diff
@@ -0,0 +1,84 @@
+{#
+/**
+ * @file
+ * Button component.
+ *
+ * @param string $label
+ * @param string $url
+ * @param array $attr
+ * @param string $icon
+ * @param string $icon_position
+ * @param 'primary'|'secondary'|'transparent' $type
+ */
+#}
+
+{# Customizable tag with a default value. #}
+{% set default_tag = url is defined or (attr is defined and attr.href is defined) ? 'a' : 'button' %}
+{% set tag = tag|default(default_tag) %}
+
+{# Default icon values #}
+{% set icon_position = icon_position|default('before') %}
+{% set icon_only = icon_only|default(false) %}
+
+{# Default attributes that could be overriden. #}
+{% set default_attributes = {
+  title: label,
+  type: tag == 'button' ? 'button' : false,
+  href: tag == 'a' and url is defined ? url : false,
```

Comment:
**suggestion**: 
Add `aria-label: icon_only ? label : false` to always have a clear name for icon buttons